account_payment_show_invoice/README.rst

```diff
@@ -1,65 +0,0 @@
-.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
a minimal README.rst file is needed for the fragments to work, isn't it? @pedrobaeza

Reply:
I don't know :S

Reply:
There are 3 options:

* Remove entirely the README, as done here, which is the less convenient, as it:

  * increases the diff, removing possible matched lines first, and then adding again by the bot
  * adds extra commit by the bot
  * provoke the error (warning) on runbot

* Leave the previous README. Better than the previous one, but it:

  * adds extra commit by the bot
  * causes possible misunderstandings seeing previous version references

* My favorite one, which is pregenerating the README in local and include it in the migration commit. For doing that:

  * clone OCA/maintainer-quality-tools
  * `sudo python3 setyp.py install` inside the dir
  * `sudo apt-get install python3-docutils`
  * go to the addon directory and put: ` oca-gen-addon-readme  --repo-name <repository> --branch <branch> --addon-dir . --no-commit`

  It's very simple once you get used to

Reply:
Wow, very helpful comment. Thanks. Why your favorite one is the more complex? 😅

Reply:
It's not very complex. It has a bit more setup, but once done, it's direct. See the cons of the other approaches in my previous comment.

Reply:
trying option 3:
first problem: there isn't setup.py nor setyp.py in that repository 🤔

Reply:
It's OCA/maintainer-tools (edited my previous comment)

Reply:
still trying option 3:
in last command, I was also required to install setuptools_scm. Done through pip3. Ok.
but I also receive the error in last command:
```
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 849, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'uritemplate.py>=0.2.0' distribution was not found and is required by github3.py
```
It's strange because when I try to install this lib through pip3, I get the message:
```
requirement already satisfied: uritemplate in /usr/local/lib/python3.5/dist-packages/uritemplate-2.0.0-py3.5.egg (2.0.0)
```
And I am blocked now with that.

Reply:
Uhm, maybe it's because this is not fully supported in Python 3. Try with Python 2 and tell me back. I don't remember how I setup it.

Reply:
same happens with python 2.7 :S

Reply:
It seems you have something strange with your PYTHONPATH, as it's not recognizing the package. Uninstall the pip version and try to install Ubuntu specific package.

Reply:
done that. still same error

Reply:
Well, if there's no way of getting that working, use option 2

Reply:
ok. done

Reply:
I run the script myself and amended the result to the last commit.